src/Microsoft.Health.Fhir.Shared.Web/appsettings.json

```diff
@@ -128,7 +128,10 @@
             "http://hl7.org/fhir/SearchParameter/individual-given",
             "http://hl7.org/fhir/SearchParameter/individual-birthdate",
             "http://hl7.org/fhir/SearchParameter/clinical-date"
-        ]
+        ],
+        "ParallelQueryOptions": {
+            "MaxQueryConcurrency": 1000
```

Comment:
@namadabu If I am not wrong, as per your queries testing using portal, 1000 is the max number for which cosmosdb sdk has returned results successfully so I am thinking if we could use max number that is known to us so far. Thoughts?

Reply:
@rbans96 for 1000 we got a much slower response compared to when using 500. So it would be better to go with 500 and then modify according to the behavior we see.